packages/loader/container-loader/src/container.ts

```diff
@@ -2586,3 +2587,17 @@ export interface IContainerExperimental extends IContainer {
 	 */
 	closeAndGetPendingLocalState?(stopBlobAttachingSignal?: AbortSignal): Promise<string>;
 }
+
+/**
+ * @deprecated Will be removed in future major release.
```

Comment:
Do you want to add more comments here for why this method was needed, and when a sufficiently distant future will be?